lib/rules/disallow-space-after-object-keys.js

```diff
@@ -7,6 +7,7 @@
  *  - `true`
  *  - `"ignoreSingleLine"` ignores objects if the object only takes up a single line
  *  - `"ignoreMultiLine"` ignores objects if the object takes up multiple lines
+ *  - `"ignoreAligned"` ignores aligned object properties
```

Comment:
We started to use `allExcept: [ "..." ]` style instead of bunch different values. Could you deprecate these values (just note in the docs is fine) while introducing the `allExcept` for this rule?

like ``allExcept: [ "aligned", "multiline", ... ]``. Also couple examples of what those values do would be cool.